Doc/library/exceptions.rst

```diff
@@ -524,6 +524,13 @@ module for more information.
    .. versionadded:: 2.5
 
 
+.. exception:: BytesWarning
+
+   Base class for warnings related to :class:`bytes` and :class:`bytearray`.
```

Comment:
In Python 2, _bytes_ is just an (undocumented?) alias for _str_. I’m not aware of BytesWarning being used in relation to Python 2’s _str_ type.